### DISCUSSION TITLE
Is this project support dead?

### DISCUSSION THREAD

**Discussion by browser-bug** [JaidedAI/EasyOCR#1229] [General]
Since we're using the tool a lot at work I was wondering if the support will continue towards the future. Judging from the high number of open issues, PRs, and pretty dated commits it looks like it. Thanks for your amazing work anyway.

**Comment by adbmdp** [comment]
Exactly… I ask myself the same question.
So many PRs to merge...

**Reply by ogencoglu** [reply]
Seems so.
